src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -812,6 +812,70 @@ def get_sorting_property(self, key: str, ids: Optional[Iterable] = None) -> np.n
         """
         return self.sorting.get_property(key, ids=ids)
 
+    def are_units_mergeable(
+        self,
+        merge_unit_groups,
+        new_unit_ids=None,
+        all_unit_ids=None,
+        merging_mode="soft",
+        sparsity_overlap=0.75,
+        new_id_strategy="append",
+        return_masks=False,
+        **kwargs,
+    ):
+        """
+        Check if merges can be performed given merging params
+
+        Parameters
+        ----------
+        merge_unit_groups : list/tuple of lists/tuples
+            A list of lists for every merge group. Each element needs to have at least two elements
+            (two units to merge).
+        sparsity_overlap : float
+            The percentage of overlap that units should share in order to accept merges.
+
+        Returns
+        -------
+        can_perform_merges : bool
+            If soft merging can be performed.
+        """
+        assert merging_mode in ["hard", "soft"], "merging_mode must be 'hard' or 'soft'"
+
+        if new_unit_ids is None:
+            new_unit_ids = generate_unit_ids_for_merge_group(
+                self.unit_ids, merge_unit_groups, new_unit_ids, new_id_strategy
+            )
```

Comment:
Why do we need `new_unit_ids` here? We can just retrun the dict whose keys are the merge unit group!

Reply:
Not sure, because this is a bit nested. new_unit_ids are needed to check what are the new unit ids, in all_unit_ids, depending on the new_id_strategy